todo/main.js

```diff
@@ -1,121 +1,125 @@
-let addingMessage = document.querySelector('.text-todo');
-let addingMessageSearch = document.querySelector('.text-search');
-// let addButton = document.querySelector('.button-plus');
-let searchInput = document.querySelector('#search');
-let toDo = document.querySelector('.todo');
-let toDoViewing = document.querySelector('.viewing');
-let done = document.querySelector('.done');
-let all = document.querySelector('.all');
-let clear = document.querySelector('.clear-all');
-let adding = document.querySelector('#adding');
+const newTaskMessage = document.querySelector(".text-todo");
+const addingButton = document.getElementById("addTask");
+const selectAll = document.querySelector(".select-all");
+const searchInput = document.querySelector("#search");
+const currentTasks = document.querySelector(".todo");
+const toDoViewing = document.querySelector("#viewing");
+const completedTasks = document.querySelector("#done");
+const fullTasksList = document.querySelector("#all");
+const clear = document.querySelector(".clear-all");
+const adding = document.querySelector("#adding");
 
 let toDoList = [];
 
 function byField(field) {
   return (a, b) => (a[field] > b[field] ? 1 : -1);
 }
 
-if (localStorage.getItem('todo')) {
-  toDoList = JSON.parse(localStorage.getItem('todo'));
+if (localStorage.getItem("todo")) {
+  toDoList = JSON.parse(localStorage.getItem("todo"));
   displayMessages();
 }
 
-function clearButts() {
-  const butts = Array.from(document.querySelector('.line').children);
-  butts.forEach((item) => {
-    item.classList.remove('active');
+function resetButtonsActivity() {
+  const allButtons = Array.from(document.querySelector(".butts").children);
+  allButtons.forEach((item) => {
```

Comment:
it should be 'enter' before and after conditions and cycles. Please check and fix it.